api/ttn/lorawan/v3/search_services.proto

```diff
@@ -341,6 +341,19 @@ message SearchAccountsRequest {
     OrganizationIdentifiers organization_ids = 6;
   }
 
+  message ExcludeFrom {
```

Comment:
Why not a `bool collaborator_conjugate` which basically says 'collaborator of' becomes 'not collaborator of' ?

Reply:
I thought it would be better to have the operation intent clear on the route but indeed nothing stops from it being just a `bool collaborator_conjugate` and the console can do `api/v3/organizations/<org_id>/collaborators/search?query=<query>&collaborator_conjugate=true`.

If you believe its best then I can remove the extra routes and leave it as boolean. Just to check but the name gives the impression that it would exclude the entity's collaborators if set to false, this would be a change in the previous behavior of the API, no?

For example:
- `api/v3/organizations/<org_id>/collaborators/search?query=<query>` currently searches for collaborators within the list of collaborators of `<org_id>`
- If we add `collaborator_conjugate`, I'm assuming that if set to `false` is when we would search for accounts outside the entity's memberships. Since the default value is `false` the base behavior of the route would change.
- This can be solved by just changing the name, so it isn't anything big, just wanted to check if I understood correctly.

Reply:
I think `conjugate` is indeed the wrong term here. I was thinking of [set complement](https://en.wikipedia.org/wiki/Set_(mathematics)#/media/File:Venn1010.svg), and I think `complement_collaborators` is maybe a better name.

Let me know what you think.

Reply:
Yeah that works better. Then I'll remove the API addition and add the `complement_collaborators`.